### PR TITLE
16 metrik export package traffic der clients

### PR DESCRIPTION
**Pull request by nbinderHHN**

### CODE REVIEW

File: metrics/package-traffic.go
Comment (nbinderHHN) [inline metrics/package-traffic.go:1]: usually files are named in snake case

File: metrics/package-traffic.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+// Record the number of bytes sent and received by a client.
+func RecordBytesSentEntry(senderClientName string, bytesSent float64) {
```
Comment (nbinderHHN) [inline metrics/package-traffic.go:9]: I would name the first param just clientName

File: metrics/package-traffic.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+func RecordBytesReceivedEntry(receiverClientName string, bytesReceived float64) {
+	bytesReceivedCounter.With(prometheus.Labels{receiverLabel: receiverClientName}).Add(bytesReceived)
+}
+
+// Maybe we should fusionate the two counters into one, with a label "direction" that can be "sent" or "received"?
+
+var bytesSentCounter = promauto.NewCounterVec(
+	prometheus.CounterOpts{Name: "bytes_sent", Help: "Number of bytes sent by a client"},
+	[]string{senderLabel})
```
Comment (nbinderHHN) [inline metrics/package-traffic.go:21]: I think the label should not be with the value "sender". It should be something simple like "client" as in this method there is not really tracked to which client the client is sending.

File: metrics/package-traffic.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+var bytesReceivedCounter = promauto.NewCounterVec(
+	prometheus.CounterOpts{Name: "bytes_received", Help: "Number of bytes received by a client"},
+	[]string{receiverLabel})
```
Comment (nbinderHHN) [inline metrics/package-traffic.go:25]: Here the same as above. I think just "client" should be better. So like this:
```suggestion
	[]string{clientLabel})
	
	const clientLabel = "client"
```

File: metrics/package-traffic.go [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+// Record the number of bytes sent and received by a client.
+func RecordBytesSentEntry(senderClientName string, bytesSent float64) {
+	bytesSentCounter.With(prometheus.Labels{senderLabel: senderClientName}).Add(bytesSent)
+}
+
+func RecordBytesReceivedEntry(receiverClientName string, bytesReceived float64) {
+	bytesReceivedCounter.With(prometheus.Labels{receiverLabel: receiverClientName}).Add(bytesReceived)
+}
+
+// Maybe we should fusionate the two counters into one, with a label "direction" that can be "sent" or "received"?
```
Comment (nbinderHHN) [inline metrics/package-traffic.go:17]: totally agree to that

**Review by Elias970 — APPROVED** [review]
all good :)

**Pull request merged**